test/config-authentication-oidc/features.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 The Knative Authors
+# Copyright 2021 The Knative Authors
```

Comment:
@creydr where is this change coming from?

Reply:
Was a copy&paste from [test/config-transport-encryption/features.yaml](https://github.com/knative/eventing/blob/main/test/config-transport-encryption/features.yaml)